src/js/structs/Item.js

```diff
@@ -7,6 +7,9 @@ module.exports = class Item {
     this._itemData = item;
   }
 
+  toJSON() {
+    return JSON.stringify(this._itemData);
```

Comment:
Do you mean `this.get('_itemData')` The whole point is to get the whole object which means also unknown values.